viewer/packages/pointclouds/visual-tests/PointCloudColorStyling.VisualTest.ts

```diff
@@ -30,7 +30,7 @@ class CustomAnnotationProvider implements PointCloudStylableObjectProvider {
     const cdfAnnotations = [
       {
         annotationId: 123,
-        region: [new Cylinder(new THREE.Vector3(-0.03, 0.1, -1000), new THREE.Vector3(-0.03, 0.1, 1000), 0.04)]
+        region: [new Cylinder(new THREE.Vector3(-0.03, 0.1, -1000), new THREE.Vector3(-0.03, 0.1, 1000), 0.04 / 1.15)]
```

Comment:
Maybe better to describe `1.15` constant with name and explain why it was added.

Reply:
I'll rather just hard code the final constant